README.md

```diff
@@ -200,6 +200,11 @@ work on your local computer, you may need to pip install it first:
 ### How can I run a notebook non-interactively?
 Under Construction.
 
+### Can I use VS code on Fornax?
+Yes!
+To open VS code on Fornax, first start up a compute session from the science console.  To do this, from the dashboard, on the left hand side menu, click on JupyterHub under Compute. 
```

Comment:
Two things, that are unrelated to this specific PR but reviewing it made me wonder:

- are we planning to regularly (e.g. yearly?) revisit the docs to make sure all these UI descriptive texts are still correct? If yes, how do we schedule that?

- Do we want to do this via screenshot? Ideally images are not great in a git repo, but I don't expect these to change much and it maybe a nicer user experience to have the visuals. It's up to you.